src/utils/toc.ts

```diff
@@ -0,0 +1,41 @@
+import slugger from 'github-slugger';
+
+interface ToCItem {
+  text: string;
+  id: string;
+  level: string;
```

Comment:
```suggestion
  level: "h1" | "h2" | "h3" | "h4" | "h5" | "h6";
```

Reply:
if changed it to number, what do you think?

Reply:
I prefer the union `h1` ... `h6` because it cannot be anything else in this situation. A number offers many other possibilities that are not possible here (negative, floating, > 6)

Reply:
what's about `1 | 2 | 3 | 4 | 5 | 6`? .. I have the feeling that the pure number is more useful for later working with the data than "h1" / "h2" ... what do you think?